tensorflow/contrib/go/doc.go

```diff
@@ -0,0 +1,6 @@
+/*
```

Comment:
About the named returned parameters (I don't know why GitHub doesn't allow me to answer directly to the named resturns comment). I usually name the returned parameters because I used to think that this was better for documenting proposals, but googling pros and cons, there are a lot of people in favour and against. Some people think that name the params causes confusion and sometimes initializes vars that don't need to be initialized and so on, but other people says that is better for auto-generated documentation.
For some methods like this:
```go
func serialize(data interface{}, deep int, dimsIn [][]int64) (ser []interface{}, dims [][]int64, dataType DataType, dataSize int64, err error)
```
I would prefer to maintain the named params. But most of the other functions can be unnamed.
Checking out some libs like [net/http](https://golang.org/pkg/net/http/) there we have some of the functions named like Client.Do, Client.Get, etc, but most of them are unnamed.

So, yes, I'm going to change almost all the methods 👍